controllers/networkpolicy/np_objects.go

```diff
@@ -0,0 +1,42 @@
+package networkpolicy
+
+import (
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func buildNetworkPolicy(ns string, desired *flowslatest.FlowCollector, additionalNs []string) (types.NamespacedName, *networkingv1.NetworkPolicy) {
+	name := types.NamespacedName{Name: constants.OperatorName, Namespace: ns}
+	if desired.Spec.NetworkPolicy.Enable == nil || !*desired.Spec.NetworkPolicy.Enable {
```

Comment:
```suggestion
	if desired.Spec.NetworkPolicy.Enable != nil || !*desired.Spec.NetworkPolicy.Enable {
```
if we want that Enable is true by default, it should be `!= nil` here